packages/fxa-auth-server/lib/senders/templates/subscriptionFailedPaymentsCancellation.html

```diff
@@ -0,0 +1,15 @@
+{{! Cancellation email has no icon }}
+{{#*inline "icon"}}
+{{/inline}}
+
+{{! This email has no title }}
+{{#*inline "title"}}
+{{/inline}}
+
+{{#*inline "content"}}
+  {{t "We've cancelled your %(productName)s subscription because multiple payment attempts failed. To get access again, start a new subscription with an updated payment method." }}
+  <br><br>
+  {{{t "Please help us improve our services by taking this <a %(cancellationSurveyLinkAttributes)s>short survey</a>." }}}
```

Comment:
I filed https://mozilla-hub.atlassian.net/browse/FXA-3717 to address this stale survey. I'd like us to remove it until we have a better alternative, but we'll see what others think.

Reply:
Leslie filed a PR to update the survey link and I reviewed and merge it.  I'll update the url here.  Thanks for catching it.

Reply:
Oh thanks for letting me know; I updated that issue (which I may close or modify depending on what Wil/Tony/Brian think). I'm glad it's updated, though we'll still need to have a survey for all products eventually or support product-specific surveys.

Reply:
err I meant rebase to update the survey url.